README.md

```diff
@@ -1,3 +1,266 @@
-### Local Environment
-Create new virtual environment and run `pip install -r dev_requirements.txt`
-Run `pre-commit install` to initialize [pre-commit](https://pre-commit.com/)
+# Neptune Scale client
+
+> [!NOTE]
+> This package only works with the `3.0` version of neptune.ai called Neptune Scale, which is in beta.
+>
+> It's supported on Linux and MacOS.
```

Comment:
Update to this.
It can be used on Windows if the run is initialized inside the `if __name__ == "__main__":` guard.

Read more here: https://docs.python.org/3/library/multiprocessing.html#multiprocessing-programming > _Safe importing of main module_

cc: @Raalsky

Reply:
It's even not specific to Windows, it's common for all of the operating systems and a common pitfall of Python itself.

Reply:
I was able to use it without the guard in Linux 🤔

Reply:
Weird, on Mac it doesn't work without the guard.

Reply:
I conclude that we don't need any OS-specific notes for now.

Reply:
To cover our bases, I'd prefer if we add a debugging/Help section of sorts and inform users to initialize the run inside  the `if __name__ == "__main__":` guard if they get the below error:

```
RuntimeError:
        An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.
```